examples/morpheusvm/tests/transfer.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package tests
+
+import (
+	"context"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/auth"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/tests/workload"
+	"github.com/ava-labs/hypersdk/tests/registry"
+
+	tworkload "github.com/ava-labs/hypersdk/tests/workload"
+	ginkgo "github.com/onsi/ginkgo/v2"
+)
+
+var _ = registry.RegisterTest("Transfer Transaction", func(t ginkgo.FullGinkgoTInterface, tn tworkload.TestNetwork) error {
+	require := require.New(t)
+
+	firstNode := tn.Nodes()[0]
+	other, err := ed25519.GeneratePrivateKey()
+	require.NoError(err)
+	aother := auth.NewED25519Address(other.PublicKey())
+
+	networkConfig := tn.Configuration().(*workload.NetworkConfiguration)
+	spendingKey := networkConfig.Keys()[0]
+
+	tx, err := firstNode.GenerateTx(context.Background(), []chain.Action{&actions.Transfer{
+		To:    aother,
+		Value: 1,
+	}},
+		auth.NewED25519Factory(spendingKey),
+	)
+	require.NoError(err)
```

Comment:
Can we move this functionality from the node to the test network interface?

We should make test writers interact with an interface that works across different environments, which will be easier if we minimize the details left to the test author like selecting which node to issue requests with.

Reply:
done